tests/functional/s3cmd/tests.js

```diff
@@ -691,22 +691,14 @@ describe('s3cmd put, get and delete object with spaces ' +
 });
 
 describe('s3cmd info', () => {
-// test that CORS and POLICY are returned as 'none'
+// test that POLICY are returned as 'none'
     it('should find that policy has a value of none', done => {
         checkRawOutput(['info', `s3://${bucket}`], 'policy', 'none',
         foundIt => {
             assert(foundIt);
             done();
         });
     });
-
-
-    it('should find that cors has a value of none', done => {
```

Comment:
I mean we should keep the s3cmd test for CORS.  You might have to skip it until the get CORS is merged but we shouldn't delete the test.  Instead convert it to show that we get the actual cors value.